ui/lib/apps/Statement/utils/tableColumns.tsx

```diff
@@ -49,7 +99,7 @@ function errorsWarningsColumn(
     name: tcf.columnName('errors_warnings'),
     key,
     fieldName: key,
-    sourceFields: ['sum_errors', 'sum_warnings'],
+    sourceFields: derivedFields.sum_errors,
```

Comment:
If we don't `getSelectedFields()` from `sourceFields` anymore, all places of `sourceFields` can be removed.

Reply:
IIRC, it's used by render function

Reply:
> IIRC, it's used by render function

Nope, it should be only used for `getSelectedFields()`.